### PR TITLE
Make the altair code a little more flexible

### PR DESCRIPTION
**Pull request by 36000**

### CODE REVIEW

File: examples/tutorial_examples/plot_001_afq_api.py [hunk 1]
```diff
@@ -204,10 +204,6 @@
 for ind in bundle_counts.index:
     if ind == "Total Recognized":
         threshold = 1000
-    elif ind == "Forceps Major":
```
Comment (arokem) [inline examples/tutorial_examples/plot_001_afq_api.py:207]: No longer an issue?

**Review by arokem — APPROVED** [review]

**Comment by arokem** [comment]
Also I'd still love to have some examples of how to use the Altair visualizations

File: examples/tutorial_examples/plot_001_afq_api.py (hunk 1 above)
Comment (36000) [reply examples/tutorial_examples/plot_001_afq_api.py:207]: We use the other callosal bundles now!

Comment (arokem) [reply examples/tutorial_examples/plot_001_afq_api.py:207]: Ah yes of course!

**Pull request merged**